shared/src/main/scala/pl/metastack/metatime/PatternTime.scala

```diff
@@ -0,0 +1,66 @@
+package pl.metastack.metatime
+
+case class PatternTime(parts: Seq[Time => String] = Nil) {
+  def string(s: String): PatternTime = add(_ => s)
+
+  def dash : PatternTime = string("-")
```

Comment:
Can you please have a look at this gist: https://gist.github.com/naqvisat/55f7cf0e21f39b1b2b62e0cde61d4826